resources/web/wwi/protoVisualizer/Vrml.js

```diff
@@ -381,7 +381,7 @@ export class SFNode extends SingleValue {
 
     // handle exceptions
     if (this.value.name === 'ImageTexture')
-      nodeX3d.setAttribute('role', parameterName.slice(0, -3)); // TODO: rename on the JS side so it matches the field name?
+      nodeX3d.setAttribute('role', parameterName.slice(0, -3));
```

Comment:
Can't we address this? If I remember right, the role of `baseColorMap` is called `baseColor` on the js side. I believe all other roles match the field name. Unless it breaks backwards compatibility or something